### ISSUE TITLE
MueLu: Uninitialized variable build warning in VariableDofLaplacianFactory

### ISSUE THREAD

**Issue by mhoemmen** [trilinos/Trilinos#1534]
@trilinos/muelu 
```
.../Trilinos/packages/muelu/src/Graph/MueLu_VariableDofLaplacianFactory_def.hpp: In member function ‘void MueLu::VariableDofLaplacianFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Build(MueLu::VariableDofLaplacianFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Level&) const [with Scalar = double; LocalOrdinal = int; GlobalOrdinal = long long int; Node = Kokkos::Compat::KokkosDeviceWrapperNode<Kokkos::Serial>; MueLu::VariableDofLaplacianFactory<Scalar, LocalOrdinal, GlobalOrdinal, Node>::Level = MueLu::Level]’:
.../Trilinos/packages/muelu/src/Graph/MueLu_VariableDofLaplacianFactory_def.hpp:249:16: warning: ‘blockRow’ may be used uninitialized in this function [-Wmaybe-uninitialized]
```

**Comment by aprokop** [comment]
Fixed in c4bea054526dd9e59c31c3df769797b9a6689492.

**Comment by mhoemmen** [comment]
Thanks!!! :D
